### ISSUE TITLE
Prepare version 1.1.7 for CRAN

### ISSUE THREAD

**Issue by mebrooks** [glmmTMB/glmmTMB#899]
@kaskr and @mmaechler we should coordinate a new version of glmmTMB to go with TMB and Matrix on CRAN. What changes do we need to make here?

We need to fix some [errors](https://cran.r-project.org/web/checks/check_results_glmmTMB.html) by April 11th to stay on CRAN. I think [this change](https://github.com/glmmTMB/glmmTMB/commit/89703b5a70e76aa4bf45369c413ec9a972cdda8e) provided by Martyn Plummer should take care of them, but I'll run the tests again early next week. If at all possible we need to do this before Friday the 7th because I will be on a plane for most of April 9th and then feeling jet-lagged on the 10th.

**Comment by bbolker** [comment]
- Can you say more about where the new CRAN errors come from (i.e. what changes in R have triggered these?) and why setting the `$modelInfo$family$dispersion` component of a model to 1 explicitly fixes the problem?
- Are we updating glmmTMB because we think the new Matrix version will break something, or because we want to avoid disturbing users with a bunch of warnings about a potentially non-compatible Matrix > TMB > glmmTMB stack?

**Comment by bbolker** [comment]
OK, now I see it: https://github.com/wch/r-source/commit/e4418eb237e2ed9db3947a432fbf69db9a4b8ca4

**Comment by mebrooks** [comment]
I haven't had a chance to dig into the details yet. I have more time on Monday. I'm back in Gainesville this weekend for a hockey tournament at UF :)

**Comment by bbolker** [comment]
Have fun!

**Comment by bbolker** [comment]
For the record, the problem is that R 4.3.0 adds a `$dispersion` element equal to 1.0 to models with fixed dispersion, where previously this value was missing/`NULL`. This only causes a problem when we load in a model fitted in a previous version of R, whose `$modelInfo$family` component thereby won't match a newly fitted object. I modified the previous fix to this so that it lives inside `up2date()`.

I updated `DESCRIPTION` and `NEWS`.

I'm currently testing with Matrix 1.5.4, so there don't seem to be any incompatibilities there.

After minimal testing (I really don't think we need to do reverse dependency checking since we haven't changed anything user-facing!)  I think we should be good to go.
